docs/advanced/theme.stories.mdx

```diff
@@ -33,7 +33,7 @@ export default function App() {
 }
 ```
 
-In most cases, the default SumUp themes provided by [@sumup/design-tokens](https://www.npmjs.com/package/sumup/design-tokens) should cover your needs. If you need to create a custom theme, we recommend overriding or making a copy of one of the [default themes](https://github.com/sumup-oss/design-tokens/blob/main/src/themes) to ensure that no values are missing.
+In most cases, the default SumUp themes provided by [@sumup/design-tokens](https://www.npmjs.com/package/@sumup/design-tokens) should cover your needs. If you need to create a custom theme, we recommend overriding or making a copy of one of the [default themes](https://github.com/sumup-oss/design-tokens/blob/main/src/themes) to ensure that no values are missing.
```

Comment:
We missed an @ in the URL here. I've checked the other links and they are correct.

The only few links that will not be accessible (yet) by those who are not org members are links to some parts of the repository tree on GitHub (because design-tokens is still private at the moment).